modules/homework-help.js

```diff
@@ -0,0 +1,77 @@
+const express = require('express');
+const db = require('../db/mysql');
+
+const router = express.Router();
+
+let errorsList = {
+    'TITLE_REQUIRED': 'Title is required',
+    'QUESTION_REQUIRED': 'Question text is required',
+    'SUBJECT_REQUIRED': 'Subject is required',
+};
+
+//category select from the DB
+let subj = {};
+
+db.query('SELECT category_name FROM category ', (err, results) => {
+    if (err) {
+        throw new Error(err);
+    }
+    subj = results;
+});
```

Comment:
nitpick: https://nodejs.org/api/modules.html#modules_caching
This place will be executed and cached on the run of the index.js file, here are the following steps to reproduce it:
1. Add console.log to debug
![image](https://user-images.githubusercontent.com/10096659/87837746-43bb7200-c89d-11ea-8b62-4c7211812767.png)

2. Start the project
```bash
MYSQL_PASSWORD='my-secret-pw' node index.js
```
3. See result:
![image](https://user-images.githubusercontent.com/10096659/87837854-b3c9f800-c89d-11ea-8227-d0c08f0fa0d8.png)